lib/rummager/config.rb

```diff
@@ -3,6 +3,16 @@
 
 configure :development do
   set :protection, false
+  set :host_authorization, {
+    permitted_hosts: [
+      "search-api.dev.gov.uk",
+      "localhost",
+      ".localhost",
+      ".test",
+      IPAddr.new("0.0.0.0/0"),
+      IPAddr.new("::/0"),
```

Comment:
(Note that these are the default hosts included in Sinatra's standard configuration)